.pylintrc

```diff
@@ -454,104 +308,76 @@ callbacks=cb_,
 
 # List of qualified module names which can have objects that can redefine
 # builtins.
-redefining-builtins-modules=six.moves,past.builtins,future.builtins,io,builtins
-
-
-[LOGGING]
-
-# Format style used to check logging format string. `old` means using %
-# formatting, while `new` is for `{}` formatting.
-logging-format-style=old
-
-# Logging modules to check that the string format arguments are in logging
-# function parameter format.
-logging-modules=logging
-
-
-[STRING]
+redefining-builtins-modules=six.moves,past.builtins,future.builtins,builtins,io
 
-# This flag controls whether the implicit-str-concat-in-sequence should
-# generate a warning on implicit string concatenation in sequences defined over
-# several lines.
-check-str-concat-over-line-jumps=no
 
+[SPELLING]
 
-[IMPORTS]
+# Limits count of emitted suggestions for spelling mistakes.
+max-spelling-suggestions=4
 
-# Allow wildcard imports from modules that define __all__.
-allow-wildcard-with-all=no
+# Spelling dictionary name. Available dictionaries: none. To make it work,
+# install the 'python-enchant' package.
+spelling-dict=
 
-# Analyse import fallback blocks. This can be used to support both Python 2 and
-# 3 compatible code, which means that the block might have code that exists
-# only in one or another interpreter, leading to false positives when analysed.
-analyse-fallback-blocks=no
+# List of comma separated words that should be considered directives if they
+# appear at the beginning of a comment and should not be checked.
+spelling-ignore-comment-directives=fmt: on,fmt: off,noqa:,noqa,nosec,isort:skip,mypy:
 
-# Deprecated modules which should not be used, separated by a comma.
-deprecated-modules=regsub,
-                   TERMIOS,
-                   Bastion,
-                   rexec
+# List of comma separated words that should not be checked.
+spelling-ignore-words=
 
-# Create a graph of external dependencies in the given file (report RP0402 must
-# not be disabled).
-ext-import-graph=
+# A path to a file that contains the private dictionary; one word per line.
+spelling-private-dict-file=
 
-# Create a graph of every (i.e. internal and external) dependencies in the
-# given file (report RP0402 must not be disabled).
-import-graph=
+# Tells whether to store unknown words to the private dictionary (see the
+# --spelling-private-dict-file option) instead of raising a message.
+spelling-store-unknown-words=no
 
-# Create a graph of internal dependencies in the given file (report RP0402 must
-# not be disabled).
-int-import-graph=
 
-# Force import order to recognize a module as part of the standard
-# compatibility libraries.
-known-standard-library=
+[STRING]
 
-# Force import order to recognize a module as part of a third party library.
-known-third-party=enchant
+# This flag controls whether inconsistent-quotes generates a warning when the
+# character used as a quote delimiter is used inconsistently within a module.
+check-quote-consistency=no
```

Comment:
What if you change this to "yes"

Reply:
It finds 299 new errors, 272 of which are from different fuzzers' `fuzzer.py` : )

Reply:
Fixed!